instructure/instructure-ui
# add jest + react-testing library with example and docs guide

INSTUI-3810

- add jest + react testing library to the project
- add rules, config and script to run them
- run with CI
- convert `ui-avatar` tests as example and skip old ones
- add guide to docs about converting tests

test plan
- run new tests with `yarn test:new` -> all 26 should pass
- run old `ui-avatar` tests with `npx ui-scripts test --path packages/ui-avatar` -> all 26 should be skipped (not including the theme related ones)
- read the guide about tests [here](https://1246--preview-instui.netlify.app/#converting-old-tests)
